src/machine/machine-varlink.h

```diff
@@ -3,4 +3,20 @@
 
 #include "sd-varlink.h"
 
+#include "machine.h"
+
+#define VARLINK_DISPATCH_MACHINE_LOOKUP_FIELDS(t) {         \
+                .name = "name",                             \
+                .type = SD_JSON_VARIANT_STRING,             \
+                .callback = sd_json_dispatch_const_string,  \
+                .offset = offsetof(t, machine_name)         \
+        }, {                                                \
+                .name = "pid",                              \
+                .type = _SD_JSON_VARIANT_TYPE_INVALID,      \
+                .callback = sd_json_dispatch_uint32,        \
+                .offset = offsetof(t, pid)                  \
+        }
```

Comment:
(btw, the cool stuff today is to support pidfds right-away)

Reply:
do you want me to implement it? It can be another input field.

Reply:
well, let me do it, when i look into coming up with a more complex generic pid structure